File: Algorithms/ReactNativeAlgorithms/__tests__/algorithms/selectSort-test.js
import selectSort from '../../src/algorithms/selectSort';

describe('selectSort', () => {
  it('returns sorted array', () => {
    var input = [0.2, 11, 3.14, -1, 1, 0.21, 10.999];
    var expected = [-1, 0.2, 0.21, 1, 3.14, 10.999, 11];
    var actual = selectSort(input);

    expect(expected).toEqual(actual);
  });
});

File: Algorithms/flutter_algorithms/test/algorithms/select_sort_test.dart
import 'package:flutter_test/flutter_test.dart';
import 'package:flutter_algorithms/algorithms/select_sort.dart';

void main() {
  test('SelectSort should properly sort an array', () {
    final List<double> input = [0.2, 11, 3.14, -1, 1, 0.21, 10.999];
    final List<double> expected = [-1, 0.2, 0.21, 1, 3.14, 10.999, 11];
    final List<double> actual = SelectSort.call(input);

    expect(expected, actual);
  });
}